xonsh/xontribs.py

```diff
@@ -225,7 +228,8 @@ def xontribs_load(
         try:
             update_context(name, ctx=ctx, full_module=full_module)
         except XontribNotInstalled:
-            bad_imports.append(name)
+            if not no_warn_missing:
```

Comment:
This is hard to understand when reading through -- it's much better if it's something like `if warn_missing:`